### ISSUE TITLE
mapgen: Add wet soil layers above stone layer

### ISSUE THREAD

**Issue by jeremyshannon** [jeremyshannon/Exile#104]
If a player lives in an area for a long time, wet soil will build up in a layer down below the surface, just above the stone layer.
 But on mapgen, none of this exists, as if the world just started. We should have mapgen add occasional wet soil sheets down there, so players can dig for a well, or for wet clay, or wet loam for planting. Probably not continous layers across a biome, but enough that the serious effort of digging down will usually be at least somewhat rewarding.

**Comment by DokimiCU** [comment]
Agree.

Patches probably make more sense than the solid wet clay layers I done on the new biomes. 

We can then get rid of some of those extra biomes that only exist to add clay?? I added those so it's possible to do large scale brick-making etc. But this would achieve the same, just to a lesser extent.

 In fact, I'm not sure why I never did this originally, I did this for the beaches...!  Wet sediment could be added in caves too, maybe.
